students/17th/team1/markkim/user/views.py

```diff
@@ -0,0 +1,89 @@
+import json
+import bcrypt
+import jwt
+
+from django.views          import View
+from django.http           import JsonResponse, HttpResponse
+
+from westagram.my_settings import SECRET
+from . models              import User
+
+
+class UserView(View):
+    def post(self, request):
+
+        try:
+            data         = json.loads(request.body)
+            print(data)
+            email        = data['email']
+            password     = data['password']
+            full_name    = data['full_name']
+            phone_number = data['phone_number']
+            username     = data['username']
+            salt         = bcrypt.gensalt()
```

Comment:
```suggestion
            salt         = bcrypt.gensalt()
```
salt를 여러번 쓰지 않기 때문에 그냥 hashpw에서 바로 선언하시고, 이 내용은 삭제해주세요.

Reply:
넵 salt삭제했어요~